plugins/kiali/DEVELOPMENT.md

```diff
@@ -2,32 +2,66 @@
 
 ## Setup
 
+### Launch script and see what happen
```

Comment:
Hi @aljesusg, we generally take a different approach to dev environments, allowing plugins to be developed independently. Can you please see if your changes/script can be adjusted to this approach?

We don't want plugins to change or even rely on `packages/` folder with the intention to later remove this folder entirely.

Instead we seek to implement standalone dev environments for individual plugins. For frontend we use following patterns for dev environments:

- With static data: https://github.com/janus-idp/backstage-plugins/blob/main/plugins/tekton/dev/index.tsx
- With data fetched from backend plugin: https://github.com/janus-idp/backstage-plugins/blob/main/plugins/ocm/dev/index.tsx
- With data fetched via proxy: https://github.com/janus-idp/backstage-plugins/blob/main/plugins/quay/dev/index.tsx


For backend plugin it can be implemented as a dev environment:
- With data fetched from a standalone local/remote deployment like here https://github.com/janus-idp/backstage-plugins/blob/main/plugins/keycloak-backend/dev/index.ts

Would that be something that you can implement or you'd be interested in?

Reply:
well. I'll take a look on this, The point is that some workmates are going to help me in kiali and I want to ake it easy for them

Reply:
well @tumido I removed that to not block this and next pr and I'll try to make the dev environment with your instructions Thanks !